_topic_map.yml

```diff
@@ -2981,6 +2981,24 @@ Topics:
   # Tracing
   - Name: Tracing requests using Jaeger
     File: serverless-tracing
+    # Custom domains
+  - Name: Mapping a custom domain name to a Knative service
+    File: serverless-domain-mapping
+    # Routes
+  - Name: Configuring routes for Knative services
+    File: serverless-configuring-routes
+# Service mesh integration
+- Name: Service Mesh integration
+  Dir: service_mesh
+  Topics:
+  - Name: Integrating Service Mesh and OpenShift Serverless with Kourier enabled
+    File: serverless-ossm-with-kourier
+  - Name: Configuring TLS for a custom domain using Service Mesh and Kourier
+    File: serverless-ossm-tls-with-kourier
+  - Name: Configuring custom domains for Knative services with Service Mesh and Kourier
+    File: serverless-ossm-custom-domains
+  - Name: Integrating Service Mesh and OpenShift Serverless without Kourier
```

Comment:
Nit: The "without Kourier" part is not really meaningful for users I believe. Maybe we should reword this to something like "Natively integrate Service Mesh and Openshift Serverless" or  such.

Reply:
I think having that without mentioning the Kourier bit for now is going to make it very confusing. We can change it when the old hacky docs get deprecated, but for now I think we should leave as is to make the distinction.